example/src/App.tsx

```diff
@@ -6,133 +6,172 @@
 
 import React, { Component } from 'react';
 import {
-  AppRegistry,
   StyleSheet,
   Text,
   View,
   ScrollView,
-  Switch,
-  NativeModules,
-  Linking,
   TouchableOpacity,
-  Platform
+  Alert,
 } from 'react-native';
 
-import {ONE_PARAMETERS} from '../config/one'
+import { ONE_PARAMETERS } from '../config/one';
+import One from 'react-native-one';
+const interaction = '/Interaction';
+const properties = { key: 'value' };
 
-const One = NativeModules.One;
-const interaction = "/Interaction";
-const properties = {key:"value"};
+class ExampleProject extends Component {
+  constructor(props: any) {
+    super(props);
+    One.enableLogging(true);
 
-class AwesomeProject extends Component {
-  
-  constructor() {
-    super();
-    One.setLogLevel(One.LogLevelAll)
-
-    // Init ONE 
+    // Init ONE
     One.init(
       ONE_PARAMETERS.siteKey,
       ONE_PARAMETERS.touchpointUri,
       ONE_PARAMETERS.apiKey,
       ONE_PARAMETERS.sharedSecret,
       ONE_PARAMETERS.userId,
       ONE_PARAMETERS.adminMode,
-      ONE_PARAMETERS.hostname
+      ONE_PARAMETERS.hostName
     );
   }
 
   _onSendInteractionWithPropertiesButtonPress = () => {
-    console.log("Sending interaction with properties...");
-    One.sendInteraction(interaction, properties).then((response) => {
-      console.log(response);
-    }, (error) => {
-      console.log(error);
```

Comment:
not sure if we care but console does have a more expressive api other than log.  You could use console.info for the success case and console.error for the failure case if it helps with the demo.

Reply:
Added, thanks!